src/Campaigns/resources/admin/components/CampaignDetailsPage/Tabs/Settings.tsx

```diff
@@ -111,8 +123,19 @@ export default () => {
 
                         <select {...register('goalType')}>
                             <option value="amount">{__('Amount raised', 'give')}</option>
-                            <option value="donation">{__('Number of Donations', 'give')}</option>
-                            <option value="donors">{__('Number of Donors', 'give')}</option>
+                            <option value="donations">{__('Number of donations', 'give')}</option>
+                            <option value="donors">{__('Number of donors', 'give')}</option>
+                            {window.GiveCampaignDetails.isRecurringEnabled && (
```

Comment:
All option components can be inside one condition, just wrap them with `fragment`. 

Example:
```js
{condition && (
  <>
    <option />
    <option />
    <option />
  </>
)}
```

Reply:
Resolved in 616b9cf7a1c68f33ee593b839d4d98de5230918c